modules/map/hdmap/adapter/xml_parser/BUILD

```diff
@@ -0,0 +1,113 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
```

Comment:
What about adding a `cc_library` named `xml_parser` here to encapsulate the details of various `xxx_xml_parser` ?
cc @xiaoxq